src/renderer.js

```diff
@@ -309,7 +309,9 @@ Renderer.prototype = {
 
     // search for hovers and other special rules for the renderer
     layers = this.processLayersRules(layers)
-
+    if (layers[0].shader.attachment === 'Map::__default__') {
```

Comment:
Wouldn't be better if you just skip them within `forEach`? This way we will just drop it if it's the first _layer_.